lib/install/validate/storage.go

```diff
@@ -44,7 +44,7 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 
 	// provide a default path if only a DS name is provided
 	if imageDSpath.Path == "" {
-		imageDSpath.Path = input.DisplayName
+		log.Debug("No path specified on datastore for image store - will use the endpointVM folder")
```

Comment:
I know I wrote this but thinking that a more pedantically correct message would be:
`No path element specified for image store - will use default`

As the validator isn't responsible for populating the path element it shouldn't be making assertions about what the value will be.